sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -3941,6 +3941,17 @@ inline pi_result piEventGetInfo(pi_event Event, pi_event_info ParamName,
   HANDLE_ERRORS(urEventGetInfo(UREvent, PropName, ParamValueSize, ParamValue,
                                ParamValueSizeRet));
 
+  if (ParamName == PI_EVENT_INFO_COMMAND_EXECUTION_STATUS) {
+    /* If the PI_EVENT_INFO_COMMAND_EXECUTION_STATUS info value is
+     * PI_EVENT_QUEUED, change it to PI_EVENT_SUBMITTED. This change is needed
+     * since sycl::info::event::event_command_status has no equivalent to
+     * PI_EVENT_QUEUED. */
+    const auto param_value_int = static_cast<pi_int32 *>(ParamValue);

```

Comment:
i dont know if we should do this here. if there's a gap in the values defined in event_command_status and in PI/UR, then this should be either handled in the caller of piEventGetInfo/urEventGetInfo, or we should just modify the values defined in the PI/UR.

Reply:
To give some context, this change was originally in OpenCL PI: https://github.com/intel/llvm/pull/9944

I moved it to pi2ur during the OpenCL port because I think it should not be part of the adapter. However, what you say also makes sense.

I had a look into the sycl source code and I'm not convinced that anything needs to be done. The function flushIfNeeded() seems to be handling this scenario (it makes sure that the event is not in QUEUED state).

So, perhaps, I should just ignore this change for the OpenCL port? Maybe @0x12CC could give some more context about why this change was needed in OpenCL PI?